src/datepicker/docs/readme.md

```diff
@@ -154,4 +154,7 @@ Depending on datepicker's current mode, the date may refer either to day, month
  * `Enter`/`Space`: Select date.
  * `Ctrl`+`Up`: Move to an upper mode.
  * `Ctrl`+`Down`: Move to a lower mode.
- * `Esc`: Will close popup, and move focus to the input.
\ No newline at end of file
+ * `Esc`: Will close popup, and move focus to the input.
+
+**Known Issues**
```

Comment:
I believe the tone of the header is wrong here - it is certainly known, but it isn't something that should be considered an issue.